apps/product/tasks.py

```
@@ -1,207 +1,983 @@
+        url = "https://api.partner.market.yandex.ru/campaigns/{campaignId}/offers/stocks"
+        headers = {
+        'Content-Type': 'application/json',
+        'Authorization': f'Bearer {api_key}'
+                }
+        response1 = requests.post(url.format(campaignId=fbs), headers=headers)
+        result1 = []
+        response2 = requests.post(url.format(campaignId=fby), headers=headers)
+        result2 = []
+        
+        while True:
+            if response1.status_code == 200 and "paging" in response1.json()["result"].keys() and "nextPageToken" in response1.json()["result"]["paging"].keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)